llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -988,15 +991,17 @@ int main(int argc, char **argv) {
   // and these declarations cause an assertion in llvm-spirv. To workaround this
   // issue remove "llvm.used" from the input module before performing any other
   // actions.
+  bool RemovedLLVMUsed = false;
   if (GlobalVariable *GV = MPtr->getGlobalVariable("llvm.used")) {
     assert(GV->user_empty() && "unexpected llvm.used users");
     GV->eraseFromParent();
+    RemovedLLVMUsed = true;
   }
```

Comment:
@sndmitriev May we move this code at the beginning of processOneModule? It wouldn't require to pass RemovedLLVMUsed down to that function.

Reply:
I think that lines 994-999 should be moved into processOneModule function.

Reply:
I agree, it makes more sense as RemovedLLVMUsed does not have to be passed. I'll make the change.